src/deploy-cromwell-on-azure/Deployer.cs

```diff
@@ -1415,6 +1416,10 @@ private async Task UpgradeBlobfuseV300Async(ConnectionInfo sshConnectionInfo)
             => await Execute("Upgrading blobfuse to 1.4.3...",
                 () => ExecuteCommandOnVirtualMachineWithRetriesAsync(sshConnectionInfo, "sudo apt-get update ; sudo apt-get --only-upgrade install blobfuse=1.4.3"));
 
+        private async Task DisableDockerServiceV300Async(ConnectionInfo sshConnectionInfo)
+            => await Execute("Disabling Docker service...",
```

Comment:
Consider adding the term "auto-start". The service itself is not disabled, but directly automatically running it upon system startup is.